client/my-sites/plugins/plugins-browser-item/style.scss

```
@@ -141,6 +141,15 @@
 	color: $studio-black;
 }
 
+.plugins-browser-item__active {
+	margin-left: 25px;
+
+	.badge {
+		border-radius: 4px;
+		font-size: 0.75rem;
```

We tend to use pre-defined variables to represent the `font-size`. You can find all of the options here:
https://github.com/Automattic/wp-calypso/blob/313df6a0173d9d5692b31cf6579957f777cf383e/packages/typography/styles/variables.scss#L38-L48

```suggestion
		font-size: $font-body-extra-small;
```